redhat-developer/devconsole-operator
# [WIP] Rewrite test/operatorsource/basic_test.go using client-go

This PR replaces the oc shell commands in OperatorSource automated test with client-go.
Steps to run this test;
`make clean`
`dep ensure -v`
`make build`
`make test-operator-source`

Signed-off-by: Pratik Jagrut <pjagrut@redhat.com>
